src/docs/ftw-availability/availability-management/index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Managing listing availability
+slug: availability-management
+updated: 2025-03-24
+category: template-availability
+ingress:
+  Learn about timezones and availability management in the Template.
+published: true
+---
+
+This article explains how listing availability is handled in the
+Sharetribe Web Template, with a focus on technical details and
+implementation. For a general overview of availability management, see
+our
+[Help Center](https://www.sharetribe.com/help/en/articles/8413212-how-availability-management-works).
+See also the
+[reference documentation for availability management](<(/references/availability)>).
+
+## Availability plan types and timezone support
+
+The API supports creating availability plans with either type
+`availabilityPlan/day` with day-level availability or type
+`availabilityPlan/time` with time range level availability. When a user
+creates a listing with the Sharetribe Web Template, the listing is
+always created with type `availabilityPlan/time`, whether the calendar
+booking type is daily, nightly, hourly or fixed.
+
+Using the `availabilityPlan/time` type means that the template can
+specify a certain time zone for the listing's availability plan, which
+is asked when creating a bookable listing. The timezone is associated
+with the availability plan and ensures that all booking times are
+interpreted according to the local time of the listing, rather than the
+time zone of the customer making the booking.
+
+However, there is a tradeoff. In order to
+[query listings](https://www.sharetribe.com/api-reference/marketplace.html#query-listings)
+that use `availabilityPlan/time`, the `availability` parameter needs to
+be set to `time-full` or `time-partial`. Queries using this
+`availability` parameter do not support pagination.
+
+By default, the template will display the first 24 results when using
+the date filter (and as pagination is not supported, the results will be
+limited to 24). This value can be modified in
+[SearchPage.duck.js file](https://github.com/sharetribe/web-template/blob/main/src/containers/SearchPage/SearchPage.duck.js#L21),
+up to a 100.
+
+![Marketplace payment flow](dates-query.png)
+
+## Availability management
+
+### Availability plan
+
+For daily and nightly listings, the template simplifies availability
+management by allowing the provider to select their daily availability.
+Under the hood, availability is automatically set from `00:00` to
```

Comment:
Would it make sense to expand the "under the hood", by e.g. linking to the function(s) that do(es) this in the template, or then adding a few sentences on how this happens?